src/migrations/hotbar-store/5.0.0-beta.10.ts

```diff
@@ -59,7 +60,7 @@ export default {
         if (workspaceHotbar?.items.length < defaultHotbarCells) {
           workspaceHotbar.items.push({
             entity: {
-              uid: cluster.id,
+              uid: createHash("md5").update(`${cluster.kubeConfigPath}:${cluster.contextName}`).digest("hex"),
```

Comment:
This is needed because `cluster.id` was previously a black box value that could not be recreated (ie uuidV4 in most cases).

Reply:
This is the line that fixes the hotbar items being disabled.